site/content/contribute/server/feature-flags.md

```diff
@@ -26,13 +26,115 @@ Feature flags allow us to be more confident in shipping features continuously to
+ - **Server** - Single Mattermost workspace
+ - **Segment** - A list of servers that can be targeted by a split
+
+### Adding a split
+
+After you've added the feature flag in code, you'll need to create a split for it.
+
+ 1. Log in to Split.
+ 2. On left sidebar, click "Splits".
+ 3. At the top click "Create split".
+ 4. Fill in the following:
+   1. Enter the name of your feature flag. It must match the name used in code in `model/feature_flags.go`.
+   2. Select "server" as the traffic type.
+   3. Optionally add tags and a description.
+   4. Click "Create".
+
+The split is now created but not targeting anything. The next section covers adding basic targeting rules.
+
+###  Add targeting rules to a split
+
+To have the split have any effect, it needs targeting rules to know which environment and servers it should set the feature flag value to. To add basic targeting rules:
+
+ 1. Find your split and click on it.
+ 2. At the top, under the name of the split select the "Environment" you want to target.
+   - See [Environments](#Environments) for a description of each.
+ 3. In the center of the screen, select "Add Rules".
+ 4. Define your treatments:
+   - Recall that treatments are just the values your feature flag can have.
+   - Boolean flags should use the default "on" and "off" treatments.
+   - For non-boolean flags, change the treatments to be the possible values. E.g. for a split that specifies a plugin's version you could have three treatments of "1.0", "1.1" and "2.0".
+ 5. Down the page a bit, set the default rule to one of your treatments.
+   - This will set the flag for all servers in the environment while you don't have any other targeting rules configured.
+ 6. At the top right, click "Save changes" and confirm them on the next screen.
+
+That covers targeting an environment with a split and adding a default rule for all servers in that environment. The next section covers targeting specific servers.
+
+### Target a specific server
+
+The ID for a "server" as seen by Split is the Mattermost server's telemetry ID. To target a specific server you need that telemetry ID. The main way to get the telemetry ID from a server would be from the analytics but you can also manually grab the telemetry ID by one of the following methods:
+
+##### Get telemetry ID via inspecting browser traffic
+
+ 1. In Chrome (any browser with dev tools will work but instructions are specific to Chrome), open the developer console and switch to the Network tab.
+ 2. Type "config" into the filter for the network requests.
+ 3. Go to the server's URL (e.g. xxxxxx.test.mattermost.cloud).
+ 4. In the network tab, click on the request for `/api/v4/config/client?format=old`.
+ 5. Click on the "Response" tab and search for "TelemetryId".
+ 6. Copy the value of "TelemetryId".
+
+##### Get telemetry ID via curl and jq
+
+```bash
+$ curl -s https://<your-workspace-url>/api/v4/config/client?format=old | jq ".TelemetryId"
+```
+#### Target using the telemetry ID
```

Comment:
```suggestion
### Target using the telemetry ID
```

Reply:
This is purposefully under the `Target a specific server` section